modules/nixos/buildbot.nix

```diff
@@ -42,6 +42,7 @@
     evalMaxMemorySize = 4096;
     evalWorkerCount = 32;
     jobReportLimit = 100;
+    outputsPath = "/var/www/buildbot/nix-outputs/";
```

Comment:
I discovered that different flakes can override each other. So I wouldn't merge this until this issue is fixed (it would be a security vulnerability.